src/battle_util.c

```diff
@@ -7058,6 +7060,22 @@ static u8 ItemEffectMoveEnd(u32 battlerId, u16 holdEffect)
             effect = ITEM_EFFECT_OTHER;
         }
         break;
+    case HOLD_EFFECT_BERSERK_GENE:
+        if (TRUE)
```

Comment:
Why is the "if" even here if it's always true?

Reply:
Leftover from when I wasn't sure if I wanted to add activation requirements in my own implementation

Reply:
Please remove them.